### DISCUSSION TITLE
How to prevent the clicked event of the element from being triggered when the move event is triggered

### DISCUSSION THREAD

**Discussion by CarbonPool** [slint-ui/slint#3540] [Q&A]

**Comment by CarbonPool** [comment]
`Element {
    moved(mouse-y, pressed-y) => {
        list.viewport-y += (mouse-y - pressed-y);
    }
    clicked => {
        ...
    }
}`

**Comment by ogoffart** [comment]
(Note: I've moved this issue to Discussions as it's better suited for general questions and community interactions. This helps keep our issue tracker focused on bug reports and feature requests)

I'm not sure i understand the question correctly, but you can possibly use the `enabled` property on TouchArea and most widgets.
you can set it to false in the callback.   
Or you can use another condition in your own clicked callback.
